include/windows/osd.h

```diff
@@ -33,7 +34,26 @@ extern "C" {
 #define BIG_ENDIAN 8765
 #define BYTE_ORDER LITTLE_ENDIAN
 
-int socketpair(int af, int type, int protocol, int socks[2]);
+/* macro used because we need access to static variable */
+#define ofi_osd_init()								\
+do{										\
+	WORD wsa_version;							\
+	WSADATA data;								\
+	int err;								\
+										\
+	wsa_version = MAKEWORD(2, 2);						\
+										\
+	err = WSAStartup(wsa_version, &data);					\
+										\
+	InitOnceExecuteOnce(&ofi_init_once, ofi_init_once_cb, &ini_lock, 0);	\
+}while(0)
+
+static inline void ofi_osd_fini()								\
```

Comment:
No need for the line continuations (`\`) for the actual function.  Argument list should be `(void)` not `()`.  This could be a real function in your `osd.c` file instead of a `static inline`.